ShemaM/My-Brand
# Ft fetch queries

#### What does this PR do?
>get queries from contact page and display them in my queries page in the admin panel
#### Description of Task to be completed
>created firestore database
>add contact form input values in the firestore
>get firestore data to be displayed in queries page
#### How should this be manually tested?
>clone this repo by: `https://github.com/ShemaM/My-Brand.git`
> checkout by : `ft-fetch-queries`
> run file `index html` go on the contact page try to fill some info in the contact form click `send`
> go on `login page `
> sign in using email `shemamugisha@protonmail.com` and password `kn88chr4!cross1`
> go on `query page` you can see the queries from the contact page
#### What are the relevant Trello story?
>https://trello.com/b/MQ9S1EBE/shemachristian
#### Screenshots (if appropriate)
>N/A
